docusaurus/docs-internals/editor-controls/data-controls/imageUpload.md

```diff
@@ -0,0 +1,218 @@
+`imageType` - describes the source of the asset, which can be one of three types: `"internal"` for images stored within the project's assets, `"external"` for images hosted on external websites, or `"unsplash"` for images sourced from the Unsplash platform. <br/>
+
+### Usage
+
+#### Basic example
+
+Standard definition with only the required keys. This control will be visible on all devices. 
+
+```js
+{
+  id: "image",
+  type: "imageUpload"
+}
+```
+
+#### Label example
+
+Adding a label on the left side of the control.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  label: "Asset"
+}
+```
+
+#### Devices examples
+
+It will be rendered on all devices. This value can be omitted since it defaults to `"all"`.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  devices: "all"
+}
+```
+
+Rendering will occur only on `desktop`.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  devices: "desktop"
+}
+```
+
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  devices: "responsive"
+}
+```
+
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  disabled: true
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" },
+      ],
+    },
+    {
+      id: "image",
+      type: "imageUpload",
+      disabled: videoType === "custom",
+    },
+  ];
+};
+```
+
+#### Config `pointer` examples
+
+The config object has a `pointer` property set to `false`. This property will not display the pointer from the uploaded image and will not allow you to locate the center of the asset.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  config: {
+    pointer: false
+  }
+}
+```
+
+
+#### Config `edit` examples
+
+The config object has an `edit` property set to `false`. This setting will not allow you to delete uploaded asset.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  config: {
+    edit: false
+  }
+}
+```
+
+
+#### Config `disableSizes` examples
+
+The config object has the `disableSizes` property set to `true`, so the top-left dropdown will not be visible.
+
+```js
+{
+  id: "image",
+  type: "imageUpload",
+  config: {
+    disableSizes: true
+  }
+}
+```
+
+#### Config `acceptedExtensions` examples
+
+The config object has an `acceptedExtensions` property set to [`"svg"`, `"png"`]. This setting will only allow you to upload these types of assets.
```

Comment:
The  `acceptedExtensions` property set to `["svg", "png"]` means that we can upload only `".svg"` or `".png"` files.


P.S. check gramatically if is ok